internal/services/cognitive/cognitive_account_rai_blocklist_resource_test.go

```diff
@@ -0,0 +1,169 @@
+package cognitive_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/cognitive/2024-10-01/raiblocklists"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type CognitiveRaiBlocklistTestResource struct{}
+
+func TestAccCognitiveRaiBlocklistSequential(t *testing.T) {
+	acceptance.RunTestsInSequence(t, map[string]map[string]func(t *testing.T){
+		"raiBlocklist": {
+			"basic":          TestAccCognitiveRaiBlocklist_basic,
+			"requiresImport": TestAccCognitiveRaiBlocklist_requiresImport,
+			"complete":       TestAccCognitiveRaiBlocklist_complete,
+			"update":         TestAccCognitiveRaiBlocklist_update,
+		},
+	})
+}
+
+func TestAccCognitiveRaiBlocklist_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_cognitive_account_rai_blocklist", "test")
+	tRaiBlocklist := CognitiveRaiBlocklistTestResource{}
```

Comment:
This is typically instantiated as `r`
```suggestion
	r := CognitiveRaiBlocklistTestResource{}
```